src/components/header/header.jsx

```diff
@@ -138,13 +140,20 @@ Header.NavLinks = ({
       link: "/",
       onClick: () => {
         openMoviePreviewModal();
-        // currently hardcoding a sound. Replace with random sound, once sounds are set up.
-        playSound(
-          "https://s3.amazonaws.com/appforest_uf/f1651255945109x881367056603119500/ohmygod.m4a"
-        );
+        playRandomSound();
       },
     },
   ];
+  const { data: soundsData, loading: soundsLoading } = useQuery(GET_SOUNDS);
+
+  function playRandomSound() {
+    if (!soundsLoading) {
+      if (soundsData?.sounds.length > 1) {
```

Comment:
nit: What if there is only 1 Sound in the db? Ideally we would play that sound, but this check would return false.